src/Apps/Collect/Components/Collection/Header/index.tsx

```diff
@@ -71,8 +71,9 @@ export class CollectionHeader extends Component<Props> {
                     mt={xs ? 0 : 3}
                     mb={3}
                     headerImageUrl={resize(collection.headerImage, {
-                      width: imageWidth * 2,
-                      height: imageHeight * 2,
+                      width: imageWidth,
+                      height: imageHeight,
```

Comment:
Perhaps you already did, but if you haven't it might be worth checking the quality of the header image in mobile and making sure it isn't degraded. I think that's the initial reason why you suggested increasing the image size here.

Reply:
Yeah that's a good point. I just checked again on mobile and the quality does go down a lot on retina displays so what we can do is only add `* 2` only for mobile

Reply:
Does this affect retina screens overall or just mobile?